### PR TITLE
ScalaDoc typo

### PR DESCRIPTION
**Pull request by lambdista**
Fix type in ScalaDoc

### CODE REVIEW

File: src/library/scala/collection/GenTraversableOnce.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 package scala
 package collection
-
+c
```
Comment (lrytz) [inline src/library/scala/collection/GenTraversableOnce.scala]: This seems to be an accidental change

**Comment by lrytz** [comment]
Hi @lambdista! Thanks for your patch.

Would you mind squashing the commits into a single one? On your branch, run `git rebase -i 2dc40cc7d7`, then `git push -f`.

**Comment by lambdista** [comment]
Hi @lrytz! Done. Let me know if something is wrong.

Thank you.

**Comment by lrytz** [comment]
Hi @lambdista, there's an still a character that was added accidentally, see here https://github.com/scala/scala/pull/4399/files#diff-5234efeb8b3c68e3ddcd215805ec5e10L11

**Comment by lambdista** [comment]
Hi @lrytz, fixed the accidental added char too and squashed into a single commit.

**Comment by lambdista** [comment]
Hi @lrytz is there any other problem? I see the build failed.

**Comment by lrytz** [comment]
/rebuild - seems to be a spurious failure. our testing infrastructure seems a bit unstable these days, we had a few of those..

**Comment by lrytz** [comment]
LGTM!

**Pull request merged**